src/scene/materials/standard-material-options-builder.js

```diff
@@ -125,6 +125,8 @@ StandardMaterialOptionsBuilder.prototype._updateMaterialOptions = function (opti
     options.lightMapFormat = stdMat.lightMap ? (stdMat.lightMap.type === TEXTURETYPE_RGBM ? 1 : (stdMat.lightMap.format === PIXELFORMAT_RGBA32F ? 2 : 0)) : null;
     options.specularAntialias = stdMat.specularAntialias && (!!stdMat.normalMap) && (!!stdMat.normalMap.mipmaps) && !isPackedNormalMap;
     options.conserveEnergy = stdMat.conserveEnergy;
+    options.opacityFadesSpecRefl = stdMat.opacityFadesSpecRefl;
```

Comment:
Any opinions about this `opacityFadesSpecRefl` naming? What about `opacityFadeSpecular`. I'm not a huge fan of cropping words in property names. @mvaligursky, you love naming debates.... 😄

Reply:
The problem is that the engine makes a distinction between specular lighting that is referred to as specular and environment map reflections referred to as reflections - and this flag affects both - so it may be unclear if environment map reflections are affected or if they are treated separately?

Reply:
Although it is true that both specular lighting and environment map reflections model specular reflections - which can be referred in a shorter form to as: specular (as I did for the PR description)

Reply:
(I just changed it to `opacityFadesSpecular` - since I'm making changes to fix other stuff in the PR anyway)

Reply:
I think we should drop 's' from 'Fades', and just have opacityFadeSpecular

Reply:
@mvaligursky approved the PR - so I'm marking this as resolved and will merge